test/itg/1.7/scenario/BO/install_and_uninstall_module.js

```diff
@@ -60,23 +61,35 @@ describe('The Install of a Module and its Uninstall', function () {
             this.client
                 .setValue(this.selector.modules_search, module_tech_name)
                 .click(this.selector.modules_search_button)
-                .waitForExist(this.selector.module_tech_name, 90000)
-                .call(done);
+                .getText(this.selector.nbr_module).then(function (text) {
+                global.nbr = text[0];
```

Comment:
It seems you never check anything regarding the test you put in it.

Did you mean `global.nbr = 0;` ?